tests/Fixtures/app/AppKernel.php

```diff
@@ -38,6 +38,14 @@ class AppKernel extends Kernel
 {
     use MicroKernelTrait;
 
+    public function __construct(string $environment, bool $debug)
+    {
+        parent::__construct($environment, $debug);
+
+        // patch for behat/symfony2-extension not supporting %env(APP_ENV)%
+        $this->environment = $_SERVER['APP_ENV'] ?? $environment;
```

Comment:
why not `getenv('APP_ENV')`?

Reply:
because it's not thread safe, and for consistency with Flex/Symfony.